MachObfuscator/Commons/Pipe.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+precedencegroup ForwardApplication {
+    associativity: left
+}
+
+infix operator |>: ForwardApplication
+
+func |> <T, U>(value: T, function: (T) -> U) -> U {
+    return function(value)
+}
```

Comment:
Perhaps we could also add function composition operator:

```
precedencegroup ForwardComposable {
    associativity: left
    higherThan: ForwardApplicable
}

infix operator >>>: ForwardComposable

public func >>><A, B, C>(_ f: @escaping (A) -> B, _ g: @escaping (B) -> C) -> (A) -> C {
    return {
        g(f($0))
    }
}

```

Reply:
Good idea. What to you think about adding this in another PR? After all this should be just migration to Swift 5 😂

Reply:
Haha right! Got overexcited!